examples/simple_flatbuffer.rs

```diff
@@ -49,11 +49,36 @@ fn get_input(packet: &GameTickPacket) -> ControllerStateArgs {
     let flat_car_rot = car_phys.rotation().expect("Missing player rotation");
     let steer = desired_yaw - flat_car_rot.yaw();
 
-    ControllerStateArgs {
+    let player_index = 0;
+    let controller_state_args = flat::ControllerStateArgs {
         throttle: 1.0,
         steer: normalize_angle(steer).max(-1.0).min(1.0),
         ..Default::default()
-    }
+    };
+
+    build_player_input(player_index, controller_state_args)
+}
+
+fn build_player_input<'a>(
```

Comment:
This function is basically the new boilerplate imposed by taking a `&[u8]` as input for the low-level functions. It's not that bad, and not nearly as bad as the `set_game_state` stuff, which is going to be less common.

Reply:
Sure, for the high-level interface it's fine to use either.